src/serac/physics/heat_transfer.hpp

```diff
@@ -502,12 +556,35 @@ class HeatTransfer<order, dim, Parameters<parameter_space...>, std::integer_sequ
     return *parameter_sensitivities_[parameter_field];
   }
 
+  /**
+   * @brief Compute the implicit sensitivity of the quantity of interest used in defining the load for the adjoint
+   * problem with respect to the shape displacement field
+   *
+   * @return The sensitivity with respect to the shape displacement
+   *
+   * @pre `solveAdjoint` with an appropriate adjoint load must be called prior to this method.
+   */
+  FiniteElementDual& computeShapeSensitivity()
+  {
+    auto drdshape = serac::get<DERIVATIVE>((*residual_)(DifferentiateWRT<2>{}, temperature_, zero_, shape_displacement_,
+                                                        *parameter_states_[parameter_indices]...));
```

Comment:
```suggestion
    auto drdshape = serac::get<DERIVATIVE>((*residual_)(DifferentiateWRT<SHAPE>{}, temperature_, zero_, shape_displacement_,
                                                        *parameter_states_[parameter_indices]...));
```

and define `SHAPE` up at the top with the other `static constexpr`s